pkgs/development/web/ping/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkgconfig
+, pantheon
+, python3
+, gtk3
+, gtksourceview
+, json-glib
+, libsoup
+, libgee
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  name = "ping-${version}";
```

Comment:
```suggestion
  pname = "ping";
```

Reply:
btw `name` defaults to `${pname}-${version}` when this attribute is present.

Reply:
Oh, I didn't know the difference between ```name``` and ```pname```, thanks !